core/templates/dev/head/pages/landing/TopicLandingPage.js

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for landing page.
+ */
+
+oppia.constant('TOPIC_LANDING_PAGE_DATA', {
+  maths: {
+    fractions: {
+      collection_id: '4UgTQUc1tala',
+      page_data: {
+        row_1: {
+          heading: 'Fractions just got easier',
+          image: 'matthew_paper.png'
+        },
+        row_2: {
+          heading: 'Fun storytelling for all',
```

Comment:
I think you don't need heading, paragraph_1, paragraph_2 to be here, right? A lot of these things are going to be common for all lessons -- so, if a piece of text is the same for all lessons, don't include it here.

Reply:
But at the same time, it provides us a way to write different messages/paragraphs based on subject or topic. (should I remove this?)

Reply:
I think it's OK to remove for text that is exactly the same.

Reply:
Moved repeated data to .html file.